### PR TITLE
Changed register page background

### LINKED ISSUE #21: Give an attractive background image to login/signup page.

### PR DESCRIPTION
**Pull request by kunal2812**
Fixes #21 

image

### CODE REVIEW

**Comment by CyrilMishra** [comment]
> Fixes #21
> 
> image

Use css to change multiple background images and try to put images in respect to Indian Tourism Places.
Currently you had used a single image.
Always remember to "Fetch Origin" first.

File: templates/home/register.html [hunk 1]
```diff
@@ -21,7 +21,12 @@
                 right: 10px;
                 z-index: 1;
                 color: aliceblue;
-            }
+            } 
+          body{
```
Comment (CyrilMishra) [inline templates/home/register.html:25]: Use css to change multiple background images and try to put images in respect to Indian Tourism Places.
Currently you had used a single image.
Always remember to "Fetch Origin" first.

**Review by CyrilMishra — CHANGES_REQUESTED** [review]
Use css to change multiple background images and try to put images in respect to Indian Tourism Places.
Currently you had used a single image.
Always remember to "Fetch Origin" first.

**Comment by kunal2812** [comment]
> 
> Use css to change multiple background images and try to put images in respect to Indian Tourism Places. Currently you had used a single image. Always remember to "Fetch Origin" first.

@CyrilMishra Fixed

**Pull request merged**